Description: use nodejs (not node) as executable
 For more info, see:
 http://lists.debian.org/debian-devel-announce/2012/07/msg00002.html
Forwarded: not-needed
Author: Marcelo Jorge Vieira <metal@alucinados.com>
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2012-12-10

--- a/bin/uglifyjs
+++ b/bin/uglifyjs
@@ -1,4 +1,4 @@
-#! /usr/bin/env node
+#! /usr/bin/nodejs
 // -*- js -*-
 
 "use strict";
--- a/test/run-tests.js
+++ b/test/run-tests.js
@@ -1,4 +1,4 @@
-#! /usr/bin/env node
+#! /usr/bin/nodejs
 
 var U = require("uglify-js");
 var path = require("path");
